src/kv/MemDB.cc

```diff
@@ -139,15 +139,23 @@ int MemDB::_init(bool create)
     if (r < 0) {
       r = -errno;
       if (r != -EEXIST) {
-        derr << __func__ << " mkdir failed: " << cpp_strerror(r) << dendl;
-        return r;
+	derr << __func__ << " mkdir failed: " << cpp_strerror(r) << dendl;
+	return r;
       }
-      return 0; // ignore EEXIST
+      r = 0; // ignore EEXIST
```

Comment:
@liewegas . The bug cause by this.  If r==EEXIST, it return and do continue. for this case, logger isn't init.